.github/renovate.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": [
+    ":preserveSemverRanges",
+    ":disableDevDependencies",
+    ":disablePeerDependencies",
+    ":semanticCommits"
+  ],
+  "enabledManagers": ["npm"],
```

Comment:
works better than yarn?

Reply:
this is a bit misleading, it's actually an opt-in for all tooling pertaining to the npm ecosystem. So it does use yarn because we have a `yarn.lock`, not a `package.lock`

Reply:
ah ok